hack/update-quarkus-platform.js

```diff
@@ -0,0 +1,124 @@
+const axios = require('axios')
+const xml2js = require('xml2js');
+const {Octokit} = require("octokit");
+const {readFile,writeFile} = require('fs/promises');
+const {spawn} = require('node:child_process');
+
+const cePomPath = "templates/quarkus/cloudevents/pom.xml"
+const httpPomPath = "templates/quarkus/http/pom.xml"
+const octokit = new Octokit({auth: process.env.GITHUB_TOKEN});
+const [owner, repo] = process.env.GITHUB_REPOSITORY.split('/')
+
+const getLatestPlatform = async () => {
+    const resp = await axios.get("https://code.quarkus.io/api/platforms")
+    return resp.data.platforms[0].streams[0].releases[0].quarkusCoreVersion
+}
+
+const prExists = async (pred) => {
+
+    let page = 1
+    const perPage = 10;
```

Comment:
Nit: Why such a small number of PRs per page? If you changed this to 50 wouldn't that potentially reduce the potential number of API calls required?

Reply:
I can decide. Larger page takes longer to get, and I assume the pr will be it top 10 PRs most of the time.